### ISSUE TITLE
/execute run [...] does not work with any of the ds commands

### ISSUE THREAD

**Issue by y2k04** [OreCruncher/DynamicSurroundingsFabric#18]
Attempting to run /execute run [ dsbiome | dsdump | dsreload | dsscript | dstod ] results in a **_Incorrect argument for command_** error at the end of the ds command.

![Screenshot 2022-01-27 083321](https://user-images.githubusercontent.com/29883287/151258364-6693e7d1-8f0c-4896-a84c-3fc2c435471c.png)

**Comment by y2k04** [comment]
This also means that the output of the command cannot be stored using the _**/execute store**_ command.

**Comment by HarvelsX** [comment]
These are client commands, they cannot be called from the server side, which makes `/execute`

**Comment by y2k04** [comment]
> These are client commands, they cannot be called from the server side, which makes `/execute`

But surely it could be adapted to be allowed on server side and require the client to have the mod as well? It would be useful for making a clock above the hotbar. I did make a data pack that used math to convert ticks to hours and minutes, but the AM/PM gets stuffed because of a programming error where it wouldn't fix itself until a certain tick in the day cycle. The problem was caused by sleeping. I found that the less than and greater than operators were a bit wack when it came to calculating those numbers. Easy fix, but annoying when you spend 8 hours writing, debugging, and improving the script.
